cloudferrylib/os/actions/vm_snapshot.py

```diff
@@ -35,10 +35,36 @@ def namespace_variable(self):
         """returns unique for cloud variable from namespace"""
         return '_'.join([VM_STATUSES, self.cloud.position])
 
-    def get_list_of_vms(self):
-        search_opts = {'all_tenants': 'True'}
-        return self.get_compute_resource().get_instances_list(
-            search_opts=search_opts)
+    def get_similar_tenants(self):
```

Comment:
Please use `dst_identity.get_tenant_id_by_name()` function to find the similar tenant, the tenant names should be case-insensitive what your code doesn't cover.